script.js

```diff
@@ -1,43 +1,53 @@
-const title = document.querySelector('#title');
-const author = document.querySelector('#author');
+const titleNew = document.querySelector('#title');
+const authorNew = document.querySelector('#author');
 const form = document.querySelector('#book-form');
 const collection = document.querySelector('#collection');
-let book;
+let book = [];
 let bookList = JSON.parse(localStorage.getItem('bookList')) || [];
-function addBook() {
-  book = {
-    title: title.value,
-    author: author.value,
-    id: Math.floor(Math.random() * 100000),
-  };
-  bookList.push(book);
-  localStorage.setItem('bookList', JSON.stringify(bookList));
-}
-function removeBook(id) {
-  bookList = bookList.filter((books) => books.id !== id);
-  localStorage.setItem('bookList', JSON.stringify(bookList));
+
+class BookClass {
+  constructor(id, title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  addBook() {
+    book = { id: this.id, title: this.title, author: this.author };
+    bookList.push(book);
+    localStorage.setItem('bookList', JSON.stringify(bookList));
+  }
+
+  removeBook() {
+    bookList = bookList.filter((books) => books.id !== this.id);
+    localStorage.setItem('bookList', JSON.stringify(bookList));
+  }
 }
+
 function populate(book) {
   const row = document.createElement('tr');
   const bookTitle = document.createElement('td');
   const bookAuthor = document.createElement('td');
   const removeBtn = document.createElement('button');
-  bookTitle.innerText = book.title;
-  bookAuthor.innerText = book.author;
+  bookTitle.innerText = `"${book.title}" by ${book.author}`;
+  // bookAuthor.innerText = book.author;
   removeBtn.innerText = 'delete';
   row.append(bookTitle, bookAuthor, removeBtn);
   collection.append(row);
   removeBtn.addEventListener('click', () => {
     removeBtn.parentElement.remove();
-    removeBook(book.id);
+    const objBookClass = new BookClass();
+    objBookClass.removeBook(book.id);
```

Comment:
- [ ] When I delete a book it is deleted from the page, but not from the local storage, so when I refresh the browser, the book still shows up. Kindly fix that. I think this may have to do with your `objBookClass` variable. If you console it, it is actually undefined so the `removeBook` method that is called on it does not really work.